examples/preview/Exporter SDK/ExportSTL/BinarySTL.cs

```diff
@@ -0,0 +1,157 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using vrcontext.walkinside.exporter;
+using System.IO;
+
+namespace ExportSimple
+{
+    class BinarySTL
+    {
+        // The scenenode reference of the Walkinside exporter SDK.
+        IntPtr m_SceneNode = IntPtr.Zero;
+        // Reference the root CAD hierarchy element in Walkinside model.
+        long m_RootBranch = -1;
+
+        /// <summary>
+        /// Initialize the walkinside exporter sdk. Making it ready to process STL geometry.
+        /// </summary>
+        /// <param name="projectpath">The folder where the walkinside model will be created.</param>
+        /// <param name="projectname">The name of the folder that contains the main walkinside project file (.vrp). </param>
+        /// <returns></returns>
+        public bool Initialize(string projectpath, string projectname)
```

Comment:
why don't you use a constructor and a Disposable pattern so that you can do:

```
using (var exporter = new BinarySTL(folder, projectname))
{
    exporter.UseColor(Color.LightGoldenrodYellow);   
    exporter.Export(d.FileName); 
}
```

Also why do you need 2 arguments here ? is projectFullPath not enough? project name could be retrieved using GetFileNameWithoutExtension.